apps/paf/fields/index.js

```diff
@@ -1031,7 +1031,7 @@ module.exports = {
     validate: ['internationalPhoneNumber', { type: 'maxlength', arguments: 20 }]
   },
   'report-person-study-email': {
-    validate: ['notUrl', 'email', { type: 'maxlength', arguments: 100 }]
```

Comment:
the notUrl validation message will also need to be removed too in validations.json

Reply:
Good catch!